src/Components/Calculator.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import Row from './RowCalcul';
+import '../css/Calc.css';
+
+const Calculator = () => {
```

Comment:
- [ ] Following the project requirements, the `Calculator.js` component should be a class-based component. Please try to refactor your component to a class-based component.

**That was not highlighted by the prior reviewer. But I strongly advise you to put that into practice. In the following project, you will refactor your component into a functional component.**